src/main/java/org/folio/rest/impl/PostGobiOrdersHelper.java

```diff
@@ -307,6 +310,34 @@ public PostGobiOrdersHelper(HttpClientInterface httpClient,
       });
   }
 
+  /**
+   * Use the contributor name type specified in mappings. If the specified type can't be found, fallback using the first one listed
+   *
+   * @param name contributor type name
+   * @return UUID of the contributor name type
+   */
+  public CompletableFuture<String> lookupContributorNameTypeId(String name) {
+    logger.info("Received contributorNameType is {}", name);
+    String query = HelperUtils.encodeValue(String.format(CQL_NAME_CRITERIA, name), logger);
+    String endpoint = String.format(CONTRIBUTOR_NAME_TYPES_ENDPOINT + QUERY + LIMIT_1, query);
+    return handleGetRequest(endpoint).thenApply(HelperUtils::extractContributorNameTypeId)
+      .thenCompose(typeId -> {
+        if (StringUtils.isEmpty(typeId)) {
+          if (DEFAULT_LOOKUP_CODE.equals(name)) {
+            logger.error("No any contributorNameType available");
```

Comment:
this is petty but please correct this to "No contributorNameTypes are available"

Reply:
Corrected